one.js

```diff
@@ -0,0 +1,34 @@
+function getTvTypes(arr) {
+  return arr.map((tv) => {
+    return tv.type;
+  });
+}
+
+function getSoldOut(arr) {
+  return arr.filter((tv) => {
+    return tv.originalStock === tv.sold;
+  });
```

Comment:
Werkt prima zo hoor, maar het kan nóg korter:
```suggestion
  return arr.filter(tv => tv.originalStock === tv.sold);
```